### ISSUE TITLE
Table is not appear in TB

### ISSUE THREAD

**Issue by truschel** [gsautter/goldengate-imagine#891]
Hi Guido

I made a individual extraction and the Table is not appear in TB. I checked the attributes of the table caption and the value "Tables startId" is missing. What do you suggest?

UUID: 7A68FFD6B40AFFE2FFFAFFEEDA1EFFB6
http://tb.plazi.org/GgServer/html/865187AEB40BFFE1FC30FC6DDF33FC2C

![image](https://user-images.githubusercontent.com/57402704/73770090-91fb0f80-475a-11ea-8a97-7043200b511a.png)
![image](https://user-images.githubusercontent.com/57402704/73770134-aa6b2a00-475a-11ea-8a89-aa997e4fcecf.png)
![image](https://user-images.githubusercontent.com/57402704/73770171-b7881900-475a-11ea-995c-82d1577904aa.png)

**Comment by gsautter** [comment]
The caption proper doesn't need a start ID, table _citations_ use that to link to the caption they cite ... the attributes of this table caption are perfectly fine.
The reason you don't see the table under "Tables" is that there is no `tableCitation` in the treatment, which makes the table appear to just be included in the treatment as a result of page layout considerations, e.g. sitting at the top or bottom of a page, and only being cited somewhere else in the text.
I checked the document now, and indeed the genus treatment doesn't cite the table it floats around, so it won't show under "Tables". However, it should show where it _is_ cited at some point in the future, that is when we get around to externalizing tables like we do for figures.

**Comment by truschel** [comment]
Hi Guido

There is no tableCitation in the treatment that I sent the print for you (name of genus). Sorry, my mistake! But the species treatments have the same tableCitation.

![image](https://user-images.githubusercontent.com/57402704/73843666-cf64a900-47fd-11ea-8459-5495b48fcfec.png)

**Comment by gsautter** [comment]
Yes, but the citing treatments do not include the table ... that's exactly why there is a plan to externalize tables just like we're doing it for figures ... only that will make the tables accessible from wherever they are cited.

**Comment by mguidoti** [comment]
Riiight. 

Sorry about that and as usual, thanks for looking into our stuff.
